<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/bulb_light.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/bulb_light.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/bulb_light.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>light1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>light2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>light3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>light4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/bulb_light.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shining">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>shining</string>
    </property>
    <property name="shortcut">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dump">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>switch on</string>
    </property>
    <property name="shortcut">
     <string>W</string>
    </property>
   </widget>
   <widget class="QLabel" name="cout">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>switching off</string>
    </property>
    <property name="shortcut">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>61</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="switchSpeed">
       <property name="text">
        <string>switchSpeed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="speedBar">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>299</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
    <property name="shortcut">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ans1</string>
    </property>
    <property name="shortcut">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ans2</string>
    </property>
    <property name="shortcut">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>650</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ans3</string>
    </property>
    <property name="shortcut">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>650</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ans4</string>
    </property>
    <property name="shortcut">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_string">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
